server/routes/logic.js

```diff
@@ -0,0 +1,123 @@
+require('dotenv').config();
+const bcrypt = require('bcrypt');
+const saltRounds = 10;
```

Comment:
place this below the imports

It's typical convention in JavaScript to have import/require statements only at the top